wgpu-types/src/instance.rs

```diff
@@ -8,15 +8,34 @@ use crate::Backends;
 use crate::{Backend, DownlevelFlags};
 
 /// Options for creating an instance.
+///
+/// If you want to have environment variable available to control instance settings, call either
```

Comment:
I think this could be clearer:

```suggestion
/// If you want to allow control of instance settings via environment variables, call either
```